SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -711,6 +716,51 @@ bool FSpatialGDKEditorToolbarModule::GenerateDefaultLaunchConfig(const FString&
 	return false;
 }
 
+bool FSpatialGDKEditorToolbarModule::GenerateDefaultWorkerJson()
+{
+	if (const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>())
+	{
+		const FString WorkerJsonPath = FSpatialGDKServicesModule::GetSpatialOSDirectory(TEXT("workers/unreal"));
```

Comment:
OutputPath?

Reply:
Renamed to WorkerJsonDir to reflect that it's the place we're looking for worker json files (and also the place we'll write one too)